styles/css/core/variables.css

```diff
@@ -1,28 +1,28 @@
-/*
- * IMPORTANT: This file is the result of assembling design tokens.
- * Do not edit directly.
- * Generated on Tue, 27 Aug 2024 17:14:44 GMT
+/**
+ * Do not edit directly, this file was auto-generated.
+ * Generated on Sat, 31 Aug 2024 18:43:09 GMT
  */
 
+
 :root {
   --pgn-theme-interval: 8%;
   --pgn-other-form-control-custom-file-content: Browse;
   --pgn-other-form-control-custom-file-lang: en;
   --pgn-other-form-control-range-track-cursor: pointer;
   --pgn-other-form-control-cursor: auto;
-  --pgn-elevation-zindex-fixed: 1030;
-  --pgn-elevation-zindex-sticky: 1020;
-  --pgn-elevation-zindex-2000: 2000;
-  --pgn-elevation-zindex-1800: 1800;
-  --pgn-elevation-zindex-1600: 1600;
-  --pgn-elevation-zindex-1400: 1400;
-  --pgn-elevation-zindex-1200: 1200;
-  --pgn-elevation-zindex-1000: 1000;
-  --pgn-elevation-zindex-800: 800;
-  --pgn-elevation-zindex-600: 600;
-  --pgn-elevation-zindex-400: 400;
-  --pgn-elevation-zindex-200: 200;
-  --pgn-elevation-zindex-0: 0;
+  --pgn-elevation-zindex-fixed: 1030; /* z-index of for fixed element. */
```

Comment:
By relying on the `formattedVariables` helper function from `style-dictionary/utils`, we get inline comments from the token's `description` field, if any. Much of the other logic we had implemented is provided out-of-the-box (e.g., ordering of CSS variables).